src/core/app.lua

```diff
@@ -178,6 +178,7 @@ function apply_config_actions (actions, conf)
       table.insert(new_app_array, app)
       app_name_to_index[name] = #new_app_array
       app.zone = zone
+      if app_table[name].start then app_table[name]:start() end
```

Comment:
I think this could be just `app` instead of `app_table[name]`; am I missing something?